### PR TITLE
Add startbootstrap-agency

### PR DESCRIPTION
**Pull request by leonciokof**

### CODE REVIEW

File: files/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "startbootstrap-agency",
+  "repo": "leonciokof/startbootstrap-agency",
+  "files": {
+    "include": ["dist/css/agency.css", "dist/css/agency.min.css", "dist/js/agency.js", "dist/js/agency.min.js"]
```
Comment (megawac) [inline files/update.json]: `"basePath": "dist"` and drop the dist in all the paths. Also can you only include the minified versions

File: files/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (megawac) [inline files/update.json]: These files belong under `/files/startbootstrap-agency` not directly under files

**Comment by leonciokof** [comment]
Sorry for errors and send a new commit with corrections

**Comment by megawac** [comment]
No worries, it happens

**Comment by leonciokof** [comment]
Approximately how long it takes the library available?

**Comment by megawac** [comment]
Hmm, it should have synced by now (it usually takes 30 minutes). I don't have access to the auto update bots logs at the moment - will have to wait for @jimaek to return on Monday. Sorry!

File: files/startbootstrap-agency/info.ini [hunk 3]
```diff
@@ -0,0 +1,5 @@
+author = "Leonardo Gatica"
+github = "https://github.com/leonciokof/startbootstrap-agency"
+homepage = "https://github.com/leonciokof/startbootstrap-agency/"
+mainfile = "A one page HTML theme for agencies created by Start Bootstrap"
+description = "dist/js/agency.min.js"
```
Comment (megawac) [inline files/startbootstrap-agency/info.ini]: This should have been `js/agency.min.js`

**Pull request merged**